### PR TITLE
Update the BLUP page

### PR DESCRIPTION
**Pull request by Elise-douard**
Adding the BLUP to the BHS website

### CODE REVIEW

**Comment by pbellec** [comment]
There are a couple broken images. I am going to look into it latter to see if I can fix it myself.

File: content/en/project/BLUP_Brain-Learning-Unicorn-Project/index.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+- Data management (feed me with multimodal data plz)
+- Statistics
+- Debugging codes
+
+# Project definition
+
+## Background
+
+<p align="center">
+<img src="Slide_content/illustration_genetic.png">
```
Comment (pbellec) [inline content/en/project/BLUP_Brain-Learning-Unicorn-Project/index.md:67]: This image is missing. It should be added in the project repo and can be referenced as ``. As long as the image is here, I'll be able to fix the link even if it's broken anyway.

File: content/en/project/BLUP_Brain-Learning-Unicorn-Project/index.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+Can a model predict the genetic profile of an individual based on brain regions volumes?
+
+## Aims
+
+- **Main goal**: 
+
+This project aims to feed a machine learning model with brain volumes to predict if an individual is carrier of a potentially pathogenic CNV.
+
+<p align="center">
+<img src="Slide_content/BLUPproject.png">
```
Comment (pbellec) [inline content/en/project/BLUP_Brain-Learning-Unicorn-Project/index.md:85]: This image is missing. It should be added in the project repo and can be referenced as ``. As long as the image is here, I'll be able to fix the link even if it's broken anyway.

**Comment by pbellec** [comment]
I actually need you to add the images I referenced to fix the broken link.

**Comment by Elise-douard** [comment]
Oups, I forgot to put the folder with the corresponding images ! I will add the folder now.

**Comment by Elise-douard** [comment]
I just added the pictures and pushed the modifications, Is it ok ?

**Comment by pbellec** [comment]
LGTM. Thanks for the submission!

**Pull request merged**